handlers/restart_consul.yml

```
@@ -6,13 +6,14 @@
   when: ansible_service_mgr == "systemd"
   listen: reload systemd daemon
 
-- name: Restart consul on Unix
-  ansible.builtin.service:
-    name: consul
-    state: restarted
-    # Needed to force SysV service manager on Docker for Molecule tests
-    use: "{{ ansible_service_mgr }}"
+- name: Rolling restart consul on Unix
+  # Manual loop on playbook hosts to perform a "serial: 1" equivalent on a set of task
+  include_tasks: ../tasks/leave_restart_consul.yml
```

```suggestion
  ansible.builtin.include_tasks: ../tasks/leave_restart_consul.yml
```

...to fix the `ansible-lint` violation.